chart/templates/cron.yaml

```
@@ -0,0 +1,62 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "k8s-ecr-login-renew.cronJob" . }}
+  labels:
+{{ include "k8s-ecr-login-renew.labels" . | indent 4 }}
+spec:
+  schedule: "* */6 * * *"
+  successfulJobsHistoryLimit: 3
+  failedJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+{{ include "k8s-ecr-login-renew.podLabels" . | indent 12 }}
+        spec:
+          serviceAccountName: {{ include "k8s-ecr-login-renew.serviceAccount" . }}
+          terminationGracePeriodSeconds: 0
+          restartPolicy: Never
+          containers:
+          - name: k8s-ecr-login-renew
+            imagePullPolicy: IfNotPresent
+            image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+            env:
+              - name: AWS_ACCESS_KEY_ID
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ include "k8s-ecr-login-renew.secret" . }}
+                    key: AWS_ACCESS_KEY_ID
+              - name: AWS_SECRET_ACCESS_KEY
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ include "k8s-ecr-login-renew.secret" . }}
+                    key: AWS_SECRET_ACCESS_KEY
```

See my previous comment

Is it possible to make the secret name + keys configurable via the values ? I like to reuse existing AWS credentials secrets across several apps like this and it's impossible if they each have a different name/key.

(don't forget to propagate to the pre-install-job as well)

```suggestion
                    name: {{ include "k8s-ecr-login-renew.secret" . }}
                    key: {{ .Values.ecr.AwsAccessKeyId_key }}
              - name: AWS_SECRET_ACCESS_KEY
                valueFrom:
                  secretKeyRef:
                    name: {{ include "k8s-ecr-login-renew.secret" . }}
                    key: {{ .Values.ecr.AwsSecretAccessKey_key }}
```